src/app/components/chefs-table/chefs-table.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="mat-elevation-z8">
+  <table mat-table class="full-width-table" matSort aria-label="Elements">
+
+    <ng-container matColumnDef="id">
+      <th mat-header-cell *matHeaderCellDef mat-sort-header>ID</th>
+      <td mat-cell *matCellDef="let row">{{row._id}}</td>
+    </ng-container>
+
+    <ng-container matColumnDef="name">
+      <th mat-header-cell *matHeaderCellDef mat-sort-header>NAME</th>
+      <td mat-cell *matCellDef="let row">{{row.name}}</td>
+    </ng-container>
+
+    <ng-container matColumnDef="description">
+      <th mat-header-cell *matHeaderCellDef mat-sort-header>DESCRIPTION</th>
+      <td mat-cell *matCellDef="let row">{{row.description}}</td>
+    </ng-container>
+
+    <ng-container matColumnDef="restaurants">
+      <th mat-header-cell *matHeaderCellDef mat-sort-header>RESTAURANTS</th>
+      <td mat-cell *matCellDef="let row">
+        <ng-container *ngIf="row.restaurants && row.restaurants.length > 0">
+          <ng-container *ngFor="let restaurant of row.restaurants; let last = last">
+            {{ restaurant.title }}
+            <span *ngIf="!last">, </span>
+          </ng-container>
+        </ng-container>
+      </td>
+    </ng-container>
+
+    <ng-container matColumnDef="chefOfTheWeek">
+      <th mat-header-cell *matHeaderCellDef mat-sort-header>CHEF OF THE WEEK</th>
+      <td mat-cell *matCellDef="let row">{{row.chefOfTheWeek}}</td>
+    </ng-container>
+
+    <ng-container matColumnDef="deleted">
+      <th mat-header-cell *matHeaderCellDef mat-sort-header>DELETED</th>
+      <td mat-cell *matCellDef="let row">{{row.deleted}}</td>
+    </ng-container>
```

Comment:
You are repeating sections 

try using ngFor and use pre set data

something like this:
<ng-container *ngFor=data in allData>
   <ng-container matColumnDef="{{data.column}}">
      <th mat-header-cell *matHeaderCellDef mat-sort-header>data.text</th>
      <td mat-cell *matCellDef="let row">{{data.row.deleted}}</td>
    </ng-container>
</ng-container>